tasks/templates/tasks/new_task.html

```diff
@@ -1,31 +1,30 @@
-{% extends 'base.html' %}
-{% block content %}
-    {% if user.is_manager %}
-        <div class="container" style="width: 60%">
-            <div class="mb-3">
-                <div class="newfrm">
-                    <h1>New Task</h1>
-                    {% if messages %}
-                        <ul class="messages">
-                            {% for message in messages %}
-                                <li class="{{ message.tags }}" style="color:{{ color }}">{{ message }}</li>
-                            {% endfor %}
-                        </ul>
-                        <br>
-                    {% endif %}
-                    <form method="POST">
-                        {% csrf_token %}
-                        <b>{{ form.as_p }}</b>
-                        <input class="btn btn-secondary btn-sm" type="submit" value="Create Task">
-                    </form>
-                </div>
-            </div>
-        </div>
-    {% elif user.user.is_authenticated %}
-        This page is restricted for managers only.
-    {% else %}
-        This page is restricted. Are you an existing user? <a href="{% url 'login' %}"> Login</a>.
-    {% endif %}
-
-{% endblock content %}
-
+{% extends 'base.html' %} {% block content %} {% if user.is_manager %}
```

Comment:
if you have a block of code that you have open "command" like: 
- "{% if user.is_manager %}" or "{% block content %}"

I expect to see the close "command" of it in the same indentation as you did in:
tasks/templates/tasks/view_single_task.html

```suggestion
{% extends 'base.html' %} 
{% block content %} 
    {% if user.is_manager %}
```

Reply:
changed by mistake please ignore